hack/iptables-wrapper-installer.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/sh
```

Comment:
This file is used to build docker image, so maybe dist/images/ is a better path to place it than hack/

Reply:
Got it.